docs/build-on-polkadot/xcm/hrmp-channels/overview.md

```diff
@@ -0,0 +1,211 @@
+
+2. Under the **configuration** section, call the `activeConfig()` function by clicking the button with the **+** icon. This will display the current settings for all the Polkadot network parameters, including the HRMP channel settings
+
+    ![](/polkadot-ecosystem-docs-draft/images/build-on-polkadot/hrmp-channels/hrmp-channels-2.webp)
+
+### Dispatching Extrinsics
+
+Establishing new HRMP channels between parachains requires dispatching specific extrinsic calls on the Polkadot relay chain from the parachain's origin.
+
+The most straightforward approach is to implement the channel opening logic off-chain, then use the XCM pallet's `send` extrinsic to submit the necessary instructions to the relay chain. However, the ability to send arbitrary programs through the `Transact` instruction in XCM is typically restricted to privileged origins, such as the sudo pallet or governance mechanisms.
+
+Parachain developers have a few options for triggering the required extrinsic calls from their parachain's origin, depending on the configuration and access controls defined:
+
+- Sudo - if the parachain has a sudo pallet configured, the sudo key holder can use the sudo extrinsic to dispatch the necessary channel opening calls
+- Governance - the parachain's governance system, such as a council or OpenGov, can be used to authorize the channel opening calls
+- Privileged Accounts - the parachain may have other designated privileged accounts that are allowed to dispatch the HRMP channel opening extrinsics
+
+## Opening HRMP Channels Between Parachains
+
+For establishing communication channels between parachains on the Polkadot network, using the HRMP protocol, the following steps are required:
+
+1. Channel request - the parachain that wants to open an HRMP channel must make a request to the parachain it wishes to have an open channel with
+2. Channel acceptance - the other parachain must then accept this request in order to complete the channel establishment
+
+This process results in a unidirectional HRMP channel, where messages can flow in only one direction between the two parachains.
+
+To enable bidirectional communication, an additional HRMP channel must be established in the opposite direction. This requires repeating the request and acceptance process, but with the parachains reversing their roles.
+
+Once both unidirectional channels are established, the parachains can then send messages back and forth freely through the bidirectional HRMP communication channel.
+
+### Procedure for Initiating HRMP Channel Setup
+
+This example will demonstrate how to open a channel between parachain 2500 and parachain 2600, using Rococo Local as the relay chain.
+
+#### Step 1 - Fund Sovereign Account {: #init-fund-sovereign-account }
+<!-- <a id="init-fund-sovereign-account"></a> -->
+
+The sovereign account for parachain 2500 on the relay chain must be funded so it can take care of any XCM transact fees.
```

Comment:
I would include a brief explanation of what a sovereign account is, or provide a reference to an external source that explains it.